app/__init__.py

```diff
@@ -41,40 +41,28 @@ def __repr__(self):
 
 
 base_url = os.getenv("URL")
-projects_base_url = base_url + "/projects/"
-profiles_base_url = base_url + "/profiles/"
+posts_base_url = base_url + "/posts/"
 
-projects = load_projects()
-profiles = load_profiles()
+posts_info = load_posts_info()
 
 
 @app.route("/")
 def index():
     return render_template(
         "index.html",
-        profiles=profiles,
-        projects=projects,
-        title="Team Kenargi's portfolio",
+        posts=posts_info,
```

Comment:
Instead of retrieving posts from the JSON file, these must be queried from the database